eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/HttpCommand.java

```diff
@@ -110,132 +114,62 @@ public HttpCommand createHttpCommandResponse(EventMeshRetCode eventMeshRetCode)
-        this.requestCode = requestCode;
-    }
-
-    public Header getHeader() {
-        return header;
-    }
-
-    public void setHeader(Header header) {
-        this.header = header;
-    }
-
-    public Body getBody() {
-        return body;
-    }
-
-    public void setBody(Body body) {
-        this.body = body;
-    }
-
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("httpCommand={")
-                .append(cmdType).append(",")
-                .append(httpMethod).append("/").append(httpVersion).append(",")
-                .append("requestCode=").append(requestCode).append(",")
-                .append("opaque=").append(opaque).append(",");
+            .append(cmdType).append(",")
+            .append(httpMethod).append("/").append(httpVersion).append(",")
+            .append("requestCode=").append(requestCode).append(",")
+            .append("opaque=").append(opaque).append(",");
 
         if (cmdType == CmdType.RES) {
             sb.append("cost=").append(resTime - reqTime).append(",");
         }
 
         sb.append("header=").append(header).append(",")
-                .append("body=").append(body)
-                .append("}");
+            .append("body=").append(body)
+            .append("}");
 
         return sb.toString();
     }
 
     public String abstractDesc() {
         StringBuilder sb = new StringBuilder();
         sb.append("httpCommand={")
-                .append(cmdType).append(",")
-                .append(httpMethod).append("/").append(httpVersion).append(",")
-                .append("requestCode=").append(requestCode).append(",")
-                .append("opaque=").append(opaque).append(",");
+            .append(cmdType).append(",")
+            .append(httpMethod).append("/").append(httpVersion).append(",")
+            .append("requestCode=").append(requestCode).append(",")
+            .append("opaque=").append(opaque).append(",");
 
         if (cmdType == CmdType.RES) {
             sb.append("cost=").append(resTime - reqTime).append(",");
         }
 
         sb.append("header=").append(header).append(",")
-                .append("bodySize=").append(body.toString().length()).append("}");
+            .append("bodySize=").append(body.toString().length()).append("}");
 
         return sb.toString();
     }
 
     public String simpleDesc() {
-        StringBuilder sb = new StringBuilder();
-        sb.append("httpCommand={")
-                .append(cmdType).append(",")
-                .append(httpMethod).append("/").append(httpVersion).append(",")
-                .append("requestCode=").append(requestCode).append("}");
-
-        return sb.toString();
+        return "httpCommand={"
+            + cmdType
+            + ","
+            + httpMethod
+            + "/" + httpVersion
+            + ","
+            + "requestCode="
+            + requestCode
+            + "}";
```

Comment:
It is recommended to use StringBuilder uniformly here to ensure the consistency of the internal code of the class as much as possible.

Reply:
Can also, with + just more brief, there is no performance gap between the two

Reply:
With no performance or code style concerns, the author should maintain consistency with the existing code.
For more details, please read the google code review guide.
https://google.github.io/eng-practices/review/reviewer/looking-for.html#consistency